src/style.css

```diff
@@ -1 +1,5 @@
-/* add styles here */
+@import './blocks/stars.css';
```

Comment:
Ensure that the `blocks/stars.css` file exists at the specified location. This is crucial for the styles to be imported and applied correctly.